.travis.yml

```diff
@@ -13,7 +13,7 @@ install:
   - export PATH=$PATH:$PWD
   - python ./manage.py collectstatic --no-input
 python:
-  - "3.6"
```

Comment:
Are we still targetting Python 3.5 as the README states ? If so, and if 3.8 is not our minimal target, I suggest that we test with travis either :

- only the lowest version we support (3.5 ?)
- all the supported python versions

This will IMHO provide better feedback about compatibility breakage, as forward compatibility is less frequent than backward compat.

Reply:
You're right, I change to add support on different version from 3.5

Reply:
Support of Python 3.5 is possible, but not testable on CI. In order to trace output of requests into the robot framework, RequestsLogger has to be used, and this script is not more compatible with python 3.5 (see https://github.com/peterservice-rnd/robotframework-requestslogger/issues/5 )

Reply:
End of security support of Python 3.5 is 2020-09-13. So I think minimal version should be 3.6.

Reply:
See #234
I separated technical upgrade and fixes in a dedicated PR.